google/cloud/spanner/value.cc

```diff
@@ -88,6 +88,77 @@ bool Equal(google::spanner::v1::Type const& pt1,
   }
 }
 
+std::ostream& StreamHelper(std::ostream& os, google::protobuf::Value const& v,
+                           google::spanner::v1::Type const& t) {
+  if (v.kind_case() == google::protobuf::Value::kNullValue &&
+      t.code() != google::spanner::v1::ARRAY &&
+      t.code() != google::spanner::v1::STRUCT) {
+    return os << "NULL";
+  }
+
+  switch (t.code()) {
+    case google::spanner::v1::BOOL:
+      return os << (v.bool_value() ? "TRUE" : "FALSE");
+
+    case google::spanner::v1::TIMESTAMP:
+    case google::spanner::v1::DATE:
+    case google::spanner::v1::INT64:
+      return os << v.string_value();
+
+    case google::spanner::v1::FLOAT64:
+      if (v.kind_case() == google::protobuf::Value::kStringValue) {
+        return os << v.string_value();
+      }
+      return os << std::to_string(v.number_value());
```

Comment:
Is this `std::to_string` needed?

Reply:
Depends. 
Without the to_string 42.0 is printed as 42 and 5.2 as 5.2.
With to_string you get 42.000000 and 5.200000

Reply:
I'd go w/ the first output, w/o the std::to_string, but I'll let you decide.

Reply:
Done.